windows/helpers/phase3/install_ci_uploader.ps1

```diff
@@ -16,7 +16,7 @@ if($isInstalled -and -not $isCurrent){
 $source="https://github.com/DataDog/datadog-ci/releases/download/v${Version}/datadog-ci_win-x64"
 $target = "c:\devtools\datadog-ci"
 
-DownloadAndExpandTo -TargetDir $target -SourceURL $source -Sha256 $Sha256
+Get-RemoteFile -LocalFile $target -RemoteFile $source -VerifyHash $Sha256
```

Comment:
I think this will download the file to `c:\devtools\datadog-ci`.
you need to make the the dest directory `c:\devtools\datadog-ci` and write it there instead.

Reply:
Thank you, just updated